layouts/_partials/sidebar-tree.html

```diff
@@ -42,7 +42,7 @@
       {{- if .Site.Params.ui.sidebar_root_enabled }} data-sidebar-root="{{ $sidebarRootID }}"{{ end -}}
     >
     {{ if and .Site.Params.ui.sidebar_lang_menu (gt (len .Site.Home.Translations) 0) -}}
-    <div class="td-sidebar-nav__section nav-item dropdown d-block d-lg-none">
+    <div class="td-sidebar-nav__section nav-item d-block d-lg-none">
       {{ partial "navbar-lang-selector.html" . }}
```

Comment:
The `.dropdown` style is applied to a div inside "navbar-lang-selector.html": https://github.com/google/docsy/blob/1b961c9b3a1999b8a294b9d0d20f13ffafb3bb4e/layouts/_partials/navbar-lang-selector.html#L3

So we don't need it on this element too.